contracts/staking/StakingInfo.sol

```diff
@@ -173,9 +173,9 @@ contract StakingInfo {
     function logSignerChange(
         uint256 validatorId,
         address oldSigner,
-        address newSigner
+        bytes memory signerPubkey
     ) public onlyStakeManager {
-        emit SignerChange(validatorId, oldSigner, newSigner);
+        emit SignerChange(validatorId, oldSigner, signerPubkey);
```

Comment:
Would be great if we could add indexed-signer as well in the event for the easier query.

Reply:
Done